tools/maybe-shim-windows.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+
+/*
+ * Adds a shim to fix Windows symlinking with create-electron-app.
+ * Should run on Windows only.
+ * More details: https://github.com/boltpkg/bolt/issues/207
+ */
+function createShim(dir, shimPath) {
+  const cwd = process.cwd();
+  fs.mkdirSync(`${cwd}/${dir}`, { recursive: true });
```

Comment:
We should use `path.resolve` instead of string concatenation, probably from `__dirname` as well instead of `cwd()`

Reply:
Oh crap, thanks, good callout - will amend this

Reply:
Couple other areas where I should do this, actually, will hit them all